[Bug]: va/va_dricommon.h: No such file or directory

### Which component impacted?

Build

### Is it regression? Good in old configuration?

Yes, it's good in old version - 22.6.2

### What happened?

Should have compiled against Libva 2.16.0

```
$ find . -name va_dricommon.h
./build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/va/x11/va_dricommon.h
```

```
BUILD      media-driver (target)
    TOOLCHAIN      cmake (auto-detect)
Executing (target): cmake -GNinja -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_TOOLCHAIN_FILE=/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/etc/cmake-x86_64-libreelec-linux-gnu.conf -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DBUILD_CMRTLIB=OFF -DBUILD_KERNELS=ON -DBUILD_TYPE=release -DENABLE_NONFREE_KERNELS=ON -DMEDIA_BUILD_FATAL_WARNINGS=ON -DMEDIA_RUN_TEST_SUITE=OFF /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/media-driver-22.6.3
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- CMAKE_INSTALL_PREFIX = /usr
-- Found PkgConfig: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config (found version "0.29.2") 
-- Could NOT find X11 (missing: X11_X11_INCLUDE_PATH X11_X11_LIB) 
-- Looking for backtrace
-- Looking for backtrace - found
-- backtrace facility detected in default set of libraries
-- Found Backtrace: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/include  
-- Checking for module 'igdgmm>=12.0.0'
--   Found igdgmm, version 12.3.0
-- media -- PLATFORM = linux
-- media -- ARCH = 64
-- media -- CMAKE_CURRENT_LIST_DIR = /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/media-driver-22.6.3/media_driver
-- media -- INCLUDED_LIBS = 
-- media -- LIB_NAME = iHD_drv_video
-- media -- OUTPUT_NAME = 
-- media -- BUILD_TYPE/UFO_BUILD_TYPE/CMAKE_BUILD_TYPE = release/release/Release
-- media -- LIBVA_INSTALL_PATH = 
-- media -- MEDIA_VERSION = 22.6.3
-- Checking for module 'libva>=1.8.0'
--   Found libva, version 1.16.0
-- LIBVA_DRIVERS_PATH = /usr/lib/dri
-- Configuring done
-- Generating done
-- Build files have been written to: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/media-driver-22.6.3/.x86_64-libreelec-linux-gnu
Executing (target): ninja 

…

[335/1467] Building CXX object media_driver/CMakeFiles/iHD_drv_v....dir/__/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp.o
FAILED: media_driver/CMakeFiles/iHD_drv_video_COMMON.dir/__/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp.o 

…

/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/media-driver-22.6.3/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp:26:10: fatal error: va/va_dricommon.h: No such file or directory
   26 | #include <va/va_dricommon.h>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
[350/1467] Building CXX object media_driver/CMakeFiles/iHD_drv_v.../media_softlet/linux/common/ddi/media_libva_interface_next.cpp.o
ninja: build stopped: subcommand failed.
```

### What's the usage scenario when you are seeing the problem?

Others

### What impacted?

with commit https://github.com/intel/media-driver/commit/1e2afc600f35d35cd2790287516541bcbd32de61 compile fails with `media-driver-22.6.3/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp:26:10: fatal error: va/va_dricommon.h: No such file or directory `

https://github.com/intel/media-driver/blob/0ba2777c8565d66065da53ea2c375b17fb4a066a/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp#L26

### Debug Information

gmmlib: update to 22.3.1
media-driver: update to 22.6.3
libva: update to 2.16.0

### Do you want to contribute a patch to fix the issue?

Yes.

Hi @Jexu - below is the `libva` we have been building. I updated the package depends to include `libxcb` - to match your suggested - but still no `va/va_dricommon.h`.

https://github.com/LibreELEC/LibreELEC.tv/blob/5132be1a1d7667c5d601e8e18cd7ed695e6e9ac2/packages/multimedia/libva/package.mk#L20-L28

```
-  PKG_DEPENDS_TARGET="toolchain libdrm"
+  PKG_DEPENDS_TARGET="toolchain libdrm libxcb"
```

### meson.log - without the addition of `libxcb`
```
libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-logs$ cat meson-log.txt 
Build started at 2022-11-22T09:09:34.199142
Main binary: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/python3
Build Options: -Dstrip=true -Ddisable_drm=false -Denable_docs=false -Denable_va_messaging=true -Dwith_x11=no -Dwith_glx=no -Dwith_wayland=no -Dprefix=/usr -Dbindir=/usr/bin -Dlibdir=/usr/lib -Dlibexecdir=/usr/lib -Dlocalstatedir=/var -Dsbindir=/usr/sbin -Dsysconfdir=/etc -Dbuildtype=plain '--cross-file /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson.conf'
Python system: Linux
The Meson build system
Version: 0.64.0
Source dir: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0
Build dir: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu
Build type: cross build
Project name: libva
Project version: 2.16.0
-----
Detecting compiler via: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc --version
compiler returned <Popen: returncode: 0 args: ['/var/media/DATA/home-rudi/LibreELEC.tv/build.L...>
compiler stdout:
x86_64-libreelec-linux-gnu-gcc-12.2.0 (GCC) 12.2.0
Copyright (C) 2022 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.


compiler stderr:

Running command: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc -E -dM -
-----
Detecting linker via: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc -Wl,--version -Wl,--as-needed -fuse-ld=gold
linker returned <Popen: returncode: 0 args: ['/var/media/DATA/home-rudi/LibreELEC.tv/build.L...>
linker stdout:
GNU gold (GNU Binutils 2.39) 1.16
Copyright (C) 2022 Free Software Foundation, Inc.
This program is free software; you may redistribute it under the terms of
the GNU General Public License version 3 or (at your option) a later version.
This program has absolutely no warranty.

linker stderr:
collect2 version 12.2.0
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/../../../../x86_64-libreelec-linux-gnu/bin/ld.gold -plugin /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/liblto_plugin.so -plugin-opt=/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/lto-wrapper -plugin-opt=-fresolution=/tmp/ccnbVESR.res -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s -plugin-opt=-pass-through=-lc -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s --sysroot=/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot --eh-frame-hdr --hash-style=gnu -m elf_x86_64 -dynamic-linker /lib64/ld-linux-x86-64.so.2 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/crt1.o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/crti.o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/crtbegin.o -L/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0 -L/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/../../../../x86_64-libreelec-linux-gnu/lib -L/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib --version --as-needed -lgcc --push-state --as-needed -lgcc_s --pop-state -lc -lgcc --push-state --as-needed -lgcc_s --pop-state /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/crtend.o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/crtn.o

Sanity testing C compiler: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc
Is cross compiler: True.
Sanity check compiler command line: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc sanitycheckc.c -o sanitycheckc_cross.exe -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -c
Sanity check compile stdout:

-----
Sanity check compile stderr:

-----
C compiler for the host machine: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc (gcc 12.2.0 "x86_64-libreelec-linux-gnu-gcc-12.2.0 (GCC) 12.2.0")
C linker for the host machine: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc ld.gold 2.39
-----
Detecting compiler via: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/host-gcc --version
compiler returned <Popen: returncode: 0 args: ['/var/media/DATA/home-rudi/LibreELEC.tv/build.L...>
compiler stdout:
gcc (Ubuntu 10.3.0-1ubuntu1~20.04) 10.3.0
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.


compiler stderr:

Running command: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/host-gcc -E -dM -
-----
Detecting linker via: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/host-gcc -Wl,--version
linker returned <Popen: returncode: 0 args: ['/var/media/DATA/home-rudi/LibreELEC.tv/build.L...>
linker stdout:
GNU ld (GNU Binutils for Ubuntu) 2.34
Copyright (C) 2020 Free Software Foundation, Inc.
This program is free software; you may redistribute it under the terms of
the GNU General Public License version 3 or (at your option) a later version.
This program has absolutely no warranty.

linker stderr:
collect2 version 10.3.0
/usr/bin/ld -plugin /usr/lib/gcc/x86_64-linux-gnu/10/liblto_plugin.so -plugin-opt=/usr/lib/gcc/x86_64-linux-gnu/10/lto-wrapper -plugin-opt=-fresolution=/tmp/cc56rgxO.res -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s -plugin-opt=-pass-through=-lc -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s --build-id --eh-frame-hdr -m elf_x86_64 --hash-style=gnu --as-needed -dynamic-linker /lib64/ld-linux-x86-64.so.2 -pie -z now -z relro /usr/lib/gcc/x86_64-linux-gnu/10/../../../x86_64-linux-gnu/Scrt1.o /usr/lib/gcc/x86_64-linux-gnu/10/../../../x86_64-linux-gnu/crti.o /usr/lib/gcc/x86_64-linux-gnu/10/crtbeginS.o -L/usr/lib/gcc/x86_64-linux-gnu/10 -L/usr/lib/gcc/x86_64-linux-gnu/10/../../../x86_64-linux-gnu -L/usr/lib/gcc/x86_64-linux-gnu/10/../../../../lib -L/lib/x86_64-linux-gnu -L/lib/../lib -L/usr/lib/x86_64-linux-gnu -L/usr/lib/../lib -L/usr/lib/gcc/x86_64-linux-gnu/10/../../.. --version -lgcc --push-state --as-needed -lgcc_s --pop-state -lc -lgcc --push-state --as-needed -lgcc_s --pop-state /usr/lib/gcc/x86_64-linux-gnu/10/crtendS.o /usr/lib/gcc/x86_64-linux-gnu/10/../../../x86_64-linux-gnu/crtn.o

Sanity testing C compiler: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/host-gcc
Is cross compiler: False.
Sanity check compiler command line: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/host-gcc sanitycheckc.c -o sanitycheckc.exe -D_FILE_OFFSET_BITS=64
Sanity check compile stdout:

-----
Sanity check compile stderr:

-----
Running test binary command:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/sanitycheckc.exe
C compiler for the build machine: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/host-gcc (gcc 10.3.0 "gcc (Ubuntu 10.3.0-1ubuntu1~20.04) 10.3.0")
C linker for the build machine: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/host-gcc ld.bfd 2.34
Build machine cpu family: x86_64
Build machine cpu: x86_64
Host machine cpu family: x86_64
Host machine cpu: x86_64
Target machine cpu family: x86_64
Target machine cpu: x86_64
Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbtxomeqa
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbtxomeqa/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbtxomeqa/output.exe -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 -Wl,--start-group -ldl -Wl,--end-group -Wl,--allow-shlib-undefined -Wl,--as-needed -fuse-ld=gold 

Code:
 int main(void) { return 0; }

Compiler stdout:
 
Compiler stderr:
 
Library dl found: YES
Pkg-config binary for 1 is not cached.
Pkg-config binary for 1 specified from cross file, native file, or env var as ['/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config']
Found pkg-config: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config (0.29.2)
Determining dependency 'libdrm' with pkg-config executable '/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config'
env[PKG_CONFIG_PATH]: 
env[PKG_CONFIG_ALLOW_SYSTEM_CFLAGS]: 1
env[PKG_CONFIG_ALLOW_SYSTEM_LIBS]: 1
env[PKG_CONFIG_LIBDIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/pkgconfig:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/share/pkgconfig
env[PKG_CONFIG_SYSROOT_BASE]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel
env[PKG_CONFIG_SYSROOT_DIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot
env[PKG_CONFIG]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config
Called `/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config --modversion libdrm` -> 0
2.4.114
env[PKG_CONFIG_PATH]: 
env[PKG_CONFIG_ALLOW_SYSTEM_CFLAGS]: 1
env[PKG_CONFIG_ALLOW_SYSTEM_LIBS]: 1
env[PKG_CONFIG_LIBDIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/pkgconfig:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/share/pkgconfig
env[PKG_CONFIG_SYSROOT_BASE]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel
env[PKG_CONFIG_SYSROOT_DIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot
env[PKG_CONFIG]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config
Called `/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config --cflags libdrm` -> 0
-I/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/include -I/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/include/libdrm
env[PKG_CONFIG_PATH]: 
env[PKG_CONFIG_ALLOW_SYSTEM_CFLAGS]: 1
env[PKG_CONFIG_ALLOW_SYSTEM_LIBS]: 1
env[PKG_CONFIG_LIBDIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/pkgconfig:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/share/pkgconfig
env[PKG_CONFIG_SYSROOT_BASE]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel
env[PKG_CONFIG_SYSROOT_DIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot
env[PKG_CONFIG]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config
Called `/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config --libs libdrm` -> 0
-L/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib -ldrm
env[PKG_CONFIG_PATH]: 
env[PKG_CONFIG_ALLOW_SYSTEM_CFLAGS]: 1
env[PKG_CONFIG_ALLOW_SYSTEM_LIBS]: 1
env[PKG_CONFIG_LIBDIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/pkgconfig:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/share/pkgconfig
env[PKG_CONFIG_SYSROOT_BASE]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel
env[PKG_CONFIG_SYSROOT_DIR]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot
env[PKG_CONFIG]: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/pkg-config
Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpra879317
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpra879317/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpra879317/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) {
            void * something;
            return 0;
        }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpra879317/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpra879317/testfile.c:4:20: warning: unused variable 'something' [-Wunused-variable]
    4 |             void * something;
      |                    ^~~~~~~~~

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpl9v7oen4
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpl9v7oen4/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpl9v7oen4/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) >= 0)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpl9v7oen4/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpl9v7oen4/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) >= 0)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpaj7hcp8p
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpaj7hcp8p/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpaj7hcp8p/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 0)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpaj7hcp8p/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpaj7hcp8p/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) > 0)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpngsxlz5q
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpngsxlz5q/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpngsxlz5q/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 1)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpngsxlz5q/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpngsxlz5q/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) > 1)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmprd2jx7t9
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmprd2jx7t9/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmprd2jx7t9/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 3)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmprd2jx7t9/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmprd2jx7t9/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) > 3)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpee32giyj
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpee32giyj/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpee32giyj/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 7)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpee32giyj/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpee32giyj/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) > 7)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpjpcuke6e
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpjpcuke6e/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpjpcuke6e/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 15)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpjpcuke6e/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpjpcuke6e/testfile.c:3:37: error: size of array 'a' is negative
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) > 15)]; a[0]=0; return 0; }
      |                                     ^
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpjpcuke6e/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmppahxwsud
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmppahxwsud/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmppahxwsud/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) <= 11)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmppahxwsud/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmppahxwsud/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) <= 11)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmp8c5ore7g
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmp8c5ore7g/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmp8c5ore7g/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) <= 9)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmp8c5ore7g/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmp8c5ore7g/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) <= 9)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbr7bulo6
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbr7bulo6/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbr7bulo6/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) <= 8)]; a[0]=0; return 0; }
Compiler stdout:
 
Compiler stderr:
 /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbr7bulo6/testfile.c: In function 'main':
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpbr7bulo6/testfile.c:3:37: warning: variable 'a' set but not used [-Wunused-but-set-variable]
    3 |         int main(void) { static int a[1-2*!(sizeof(void *) <= 8)]; a[0]=0; return 0; }
      |                                     ^

Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpfzuy3rnv
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpfzuy3rnv/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpfzuy3rnv/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 --print-search-dirs 

Code:
 
Compiler stdout:
 install: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/
programs: =/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/../../../../x86_64-libreelec-linux-gnu/bin/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/../../../../x86_64-libreelec-linux-gnu/bin/
libraries: =/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/../../../../x86_64-libreelec-linux-gnu/lib/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/lib/gcc/x86_64-libreelec-linux-gnu/12.2.0/../../../../x86_64-libreelec-linux-gnu/lib/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/lib/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/lib/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/x86_64-libreelec-linux-gnu/12.2.0/:/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/lib/

Compiler stderr:
 
Run-time dependency libdrm found: YES 2.4.114
Running compile:
Working directory:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpga8tc15_
Command line:  /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpga8tc15_/testfile.c -o /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson-private/tmpga8tc15_/output.obj -c -march=x86-64-v3 -Wall -pipe -O2 -fomit-frame-pointer -DNDEBUG -D_FILE_OFFSET_BITS=64 -O0 -fvisibility=hidden 

Code:
 extern int i;
int i;

Compiler stdout:
 
Compiler stderr:
 
Compiler for C supports arguments -fvisibility=hidden: YES 
Configuring va_version.h using configuration
Program doxygen found: NO
Build targets in project: 2
NOTICE: Future-deprecated features used:
 * 0.64.0: {'copy arg in configure_file'}

libva 2.16.0

  User defined options
    Cross files        : /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/.x86_64-libreelec-linux-gnu/meson.conf
    bindir             : /usr/bin
    buildtype          : plain
    libdir             : /usr/lib
    libexecdir         : /usr/lib
    localstatedir      : /var
    prefix             : /usr
    sbindir            : /usr/sbin
    strip              : true
    sysconfdir         : /etc
    disable_drm        : false
    enable_docs        : false
    enable_va_messaging: true
    with_glx           : no
    with_wayland       : no
    with_x11           : no

Found ninja-1.11.1 at /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/bin/ninja
```

More testing:

```
cp build.LibreELEC-Generic.x86_64-11.0-devel/build/libva-2.16.0/va/x11/va_dricommon.h \
   build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/include/va
…
## compiled now fails with `X11/Xlib.h`
- so looking like x11/va_dricommon.h is not appropriate for non X11 builds
…
In file included from /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/build/media-driver-22.6.3/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp:26:
/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-11.0-devel/toolchain/x86_64-libreelec-linux-gnu/sysroot/usr/include/va/va_dricommon.h:28:10: fatal error: X11/Xlib.h: No such file or directory
   28 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
compilation terminated.
```
So is the correct fix as below?

```
#if defined(X11_FOUND)
#include <va/va_dricommon.h>
#endif 
```

--- a/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp
+++ b/media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp
@@ -23,7 +23,6 @@
 //! \file     ddi_vp_functions.cpp
 //! \brief    ddi vp functions implementaion.
 //!
-#include <va/va_dricommon.h>
 #include "ddi_vp_functions.h"
 #include "media_libva_util_next.h"
 #include "media_libva_common_next.h"

